api/prisma/seed-helpers/application-factory.ts

```diff
@@ -45,7 +46,9 @@ export const applicationFactory = async (optionalParams?: {
     applicant: { create: applicantFactory(optionalParams?.applicant) },
     appUrl: '',
     status: ApplicationStatusEnum.submitted,
-    submissionType: ApplicationSubmissionTypeEnum.electronical,
+    submissionType: optionalParams?.submissionType
```

Comment:
you can change this to `optionalParams?.submissionType ?? ApplicationSubmissionTypeEnum.electronical`